modules/motions/utils/checkInputsLargeThenLimit.ts

```diff
@@ -0,0 +1,22 @@
+type InputValue = {
+  address: string
+  amount: string
+}
+
+export function checkInputsLargeThenLimit({
```

Comment:
If I understood purpose of this fn correctly, then `checkInputsGreaterThanLimit` will better fit linguistic perception, imo.